examples/Baselines/Halite_competition/paddle/requirements.txt

```diff
@@ -0,0 +1,4 @@
+parl
```

Comment:
Please specify the version requirement (e.g. parl>=2.0.0)

Reply:
fixed